files/zh-cn/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 实验性、已弃用和过时
+slug: MDN/Writing_guidelines/Experimental_deprecated_obsolete
+---
+
+{{MDNSidebar}}
+
+这些术语通常与技术和规范相关联，还用于在 MDN Web 文档上标记技术的状态。对于这些术语的定义，MDN Web 文档与[浏览器兼容性数据（BCD）](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#status-information)仓库保持一致。下文描述了这些术语以及它们应在 MDN Web 文档的哪些上下文中使用。
+
+## 实验性
+
+当一项技术在 MDN Web 文档中被描述为实验性（experimental）时，这意味着该技术是新出现且不成熟的，目前*正在*被添加到 web 平台（或正在考虑添加）。将一项技术标记为实验性表明读者在任何生产项目（即不仅仅是演示或实验的项目）中使用该技术之前应该谨慎考虑。鼓励读者[尝试实验性特性](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#status-information)并向浏览器供应商和标准作者提供反馈。
+
+如果适用以下一项或多项条件，则将其标记为**实验性**的技术：
+
+- 它**仅在一个**现代主流浏览器渲染引擎的发行版本中被默认实现和启用。
+- 它仅通过更改配置（如首选项或标志）而被支持，无论支持的渲染引擎数量。
+- 它的定义规范可能会以不兼容的方式发生重大变化（即，它可能会破坏依赖该特性的现有代码）。
+
+> **备注：** 仅在一个渲染引擎上发布的特性，仍被视为是实验性的，即使在其他渲染引擎的预览版本中也可用（或通过设置首选项或标志）。
+
+如果满足以下一项或多项条件，则技术可能不再是**实验性**状态：
+
+- 它在**两个或更多**主流浏览器渲染引擎中被默认支持。
+- 它在一个浏览器渲染引擎中默认支持两年或更长时间，并且没有发生重大变化。
+- 它的定义规范不太可能以破坏兼容性的方式发生变化。
+
+有关这些条件的示例，请参阅 BCD 文档：[实验性标志](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental)。
+
+通常，如果一项技术被多个主流浏览器支持，则其规范将变成稳定的，但并非总是如此。另一方面，有些技术可能有一个稳定的规范，但没有被浏览器原生支持。例如，[IMSC](/zh-CN/docs/Related/IMSC) 需要通过 JavaScript polyfill 使用。
+
+作为现行规范或标准化流程的一部分且未标记为**已弃用**的特性或技术被认为处于**标准跟踪**（standard track）状态。
```

Comment:
see https://dorado.w3.org/category/w3c-rec/4/

![image](https://github.com/mdn/translated-content/assets/15844309/387ba0c0-aa62-48f9-8a0f-fb39f01cae39)

After doing a search for related terms, I recommend the translation: "动态标准".

See also https://www.w3.org/2020/10/TPAC/w3c_2020_living_standards_and_reviews.html:

> Some will say we don't need Recommendation because we're doing a pure **living standards**, our spec is changing all the time. And yes, some part of the spec are stable, some parts are unstable, and it's very difficult to pinpoint a point in time where the spec is actually stable as a whole. A good example of that has been the Service Worker specification that we've been struggling for the past two or three years to pinpoint a point where in time where we can say, "Okay, this is a spec now, let's make it a Recommendation." Despite the fact that we have all of the testing and everything, SVG has been long running - SVG 2, we'll probably - with the level of resources we unfortunately have in the Working Group - we probably won't have, won't issue a Recommendation of it at the moment.

Reply:
I prefer '动态标准' here (towards original text 'active specification or standardization') 👍

Reply:
不过，living standard译为“现行标准”貌似也能说通

living和“现行”也都有“正在施行，正在生效（currently in effect）”的意思

Reply:
这里应该更加强调“目前的标准不是固定的”，例如 RFC、CSS V3.1，这些规范均以特定版本号固定，并非随着技术的发展而一直动态更新。“现行标准”这一说法更偏向于我们目前正在实施某一个版本的规范。W3C 在上述的声明中提到了“our spec is changing all the time”，这句话也是对前文中“pure living standards”的解释。而现如今，WHATWG 的 DOM 规范也的确不再提供具体的版本号：https://dom.spec.whatwg.org/